lms/static/js/spec/student_account/access_spec.js

```diff
@@ -149,8 +149,11 @@ define([
                 // Simulate a click on the reset password link
                 view.resetPassword();
 
-                // Verify that the password reset wrapper is populated
-                expect($('#password-reset-wrapper')).not.toBeEmpty();
```

Comment:
`$('#password-reset-wrapper')` selector is not exited in the DOM hence calling `expect($('#password-reset-wrapper')).not.toBeEmpty();` always returns `True`

If an element is not present in the DOM & You run `$('selector').is(':empty')` it returns `False` and with `.not` it is returning `True`. (Behind the scenes `toBeEmpty()` is uses `is(':empty')` )

When I searched for this elector in the `Edx-Platform` i could not find its occurrence other then in this file.